easybuild/easyconfigs/k/king/king-2.2.4.eb

```
@@ -0,0 +1,29 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'BinariesTarball'
+
+name = 'king'
+version = '2.2.4'
+local_version_shorthand = '224'
+
+homepage = 'https://people.virginia.edu/~wc9c/KING/'
+description = """KING is a toolset that makes use of high-throughput 
+SNP data typically seen in a genome-wide association study (GWAS) or 
+a sequencing project. Applications of KING include family relationship 
+inference and pedigree error checking, quality control, population 
+substructure identification, forensics, gene mapping, etc."""
+
+toolchain = {'name': 'system', 'version': ''}
+
+source_urls = ['https://people.virginia.edu/~wc9c/KING/executables']
+sources = ['Linux-%%(name)s%s.tar.gz' % (local_version_shorthand,)]
```

I would simplify the shorthand version specification in sources by just removing the dots from `version`
```suggestion
sources = ['Linux-%%(name)s%s.tar.gz' % version.replace('.', '')]
```

---

Perfect! This was what I was looking for when I asked about it in Slack!